lib/src/appengine_internal.dart

```diff
@@ -178,6 +179,12 @@ Future<ContextRegistry> _initializeAppEngine() async {
     dbEmulatorHost,
   );
 
+  // Setup a default authClient for package:gcloud
+  gcloud_http.registerAuthClientService(
+      await auth.clientViaApplicationDefaultCredentials(
+    scopes: ['https://www.googleapis.com/auth/cloud-platform'],
```

Comment:
Small nit: I haven't found any `*Scope.<value>` definition in `package:gcloud`, if there is any that you are aware of, let's use that for this value.

Reply:
they exist in `package:googleapis`, each API seems to define their own set of scopes..

I'm not exactly sure if `https://www.googleapis.com/auth/cloud-platform` will allow us to do everything, or just some things.

See: https://developers.google.com/identity/protocols/oauth2/scopes

I suspect this scope is just a catch all for access to any GCP resource (modulo any IAM policies / roles), but I'm not 100% sure how to interpret it.